src/content/getting-started/prepare-your-provider-account/aws/_index.md

```diff
@@ -2,11 +2,212 @@
+
+It's crucial that the __Require external ID__ and __Require MFA__ options remain unchecked!
+
+Then, please go ahead and proceed to the next step.
+
+#### 3. Attach all policies to role {#iam-capa-controller-role-policy}
+
+Now, you must go through all the policies created before and attach them to the node. Search for the installation name, select the policies and hit the __Next__ button.
+
+![AWS IAM console: Attach policies](aws-roles-attach-policy.png)
+
+#### 4. Name the role {#iam-capa-controller-role-name}
+
+The last step of role creation requires you to set a name for the role. Same as with the policies, please use the name changing the INSTALLATION_NAME placeholder to the name of your installation.
+
+```nohighlight
+giantswarm-${INSTALLATION_NAME}-capa-controller
+```
+
+![AWS IAM console: Review](aws-roles-review.png)
+
+__Note__: The role name also contains the installation name to make easier identifying across other roles and avoid conflicts with other installations running within the same account.
+
+### Staff permissions {#iam-staff-role}
+
+Finally, we create an IAM role for Giant Swarm support staff to assume in order to
+access both AWS accounts. This role must have Giant Swarm's account as a trusted
+entity, and we recommend that it enforces multi-factor authentication.
+
+Giant Swarm staff require access to all accounts, so the following steps must be duplicated in both the management cluster and workload cluster accounts.
+
+#### 1. Create the admin policy {#iam-staff-policy}
+
+Go to the [Policies](https://console.aws.amazon.com/iam/home#/policies) subsection and select __Create policy__ to set the admin permissions. Use the [admin JSON policy](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/raw/master/admin-role/iam-policy.json) file as the policy content. This time, call the policy
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+#### 2. Create the admin role {#iam-staff-role}
+
+Enter again in the [Roles](https://console.aws.amazon.com/iam/home#/roles) subsection of the AWS console and select __Create role__ When asked to __Select type of trusted entity__ choose __AWS account__.
+
+- In __Account ID__ enter the value `084190472784`.
+- Don't enable __Require external ID__.
+- It 's strongly recommended to check the option __Require MFA__ (multi factor authentication). This adds an extra authentication step for users to assume the role, which increases security.
+
+#### 3. Attach policy to role {#iam-staff-role-policy}
+
+Attach the newly created `GiantSwarmAdmin` policy to the role you are creating.
+
+#### 4. Name the role {#iam-staff-role-name}
+
+Name this role:
+
+```nohighlight
+GiantSwarmAdmin
+```
+
+## Step 3: Configure the cluster role identity {#configure-cluster-role-identity}
+
+Remember that `default` AWSClusterRoleIdentity is needed but you can additional add more with specific names. (LETS INCLUDE THIS IN CAPZ TOO)
```

Comment:
Not reviewing this part as it's unfinished. We first have to explain what `AWSClusterRoleIdentity` is, specifically for CAPA, e.g. how the `default` one defines in which AWS account clusters will be created unless otherwise specified.